package.json

```diff
@@ -4,13 +4,13 @@
   "description": "libp2p-webrtc-star without webrtc. Just plain socket.io.",
   "main": "src/index.js",
   "bin": {
-    "websockets-star": "src/rendezvous/bin.js",
+    "websockets-star": "src/rendezvous/bin.js"
```

Comment:
What about #11 ?

Reply:
answered in the issue Note, websocket-star doesn't do signaling, it is actually relaying